stages/infra-setup/TCID-DIR-INSTALL-ON-LOCAL-HP

```diff
@@ -53,21 +55,48 @@ kubectl create secret docker-registry directoronprem-registry-secret --docker-se
 # Create clusterrolebinding
 kubectl create clusterrolebinding kube-admin --clusterrole cluster-admin --serviceaccount=kube-system:default
 
-helm repo update
+# Deploy Kubera using either kubera-charts repo or Kubera official charts
+
+if [ "$USE_KUBERA_REPO" == "true" ]
+then
+
+  # Deploy Kubera from kubera-charts repo
+  echo -e "\nDeploying Kubera using kubera-charts repo\n"
+  echo -e "\nCloning $KUBERA_BRANCH branch of kubera-charts repo\n"
+  git clone -b $KUBERA_BRANCH https://$GITHUB_USERNAME:$GITHUB_PASSWORD@github.com/mayadata-io/kubera-charts.git
+  cd kubera-charts
+
+  echo -e "\nUpdating values in kubera-charts/values.yaml\n"
+  # Update release version
+  sed 's|release: .*|release: '$RELEASE'|' -i ./values.yaml
+  # Update URL for accessing Kubera
+  sed 's|url: .*|url: '$DOP_URL'|' -i ./values.yaml
+  # Update max member count in a project to 100
+  sed 's|maxMemberCountInOneProject: .*|maxMemberCountInOneProject: 100|' -i ./values.yaml
+  # Enable openebs RC builds deployment
+  sed 's|openebsRCEnable: .*|openebsRCEnable: true|' -i ./values.yaml
+  # Disable nginx-ingress deployment as it is predeployed in Rancher
+  ## Sed command to replace the third line next to "nginx-ingress:"
+  sed '/nginx-ingress:/!b;n;n;n;c\  enabled: false' -i ./values.yaml
+  echo -e "\nInstalling Kubera using kubera-charts/values.yaml"
+  helm install --name kubera .
```

Comment:
@harshshekhar15 I have a very basic query:
On which machine is this command run?
- Is this done from a gitlab machine?
- Is this done from the lab machine (landing)?
- Is this done from a K8s node?